src/TransitionGroup.js

```diff
@@ -6,16 +6,19 @@ import { assign, linkRef } from './util';
 const identity = i => i;
 
 export class TransitionGroup extends Component {
-	static defaultProps = {
-		component: 'span',
-		childFactory: identity
-	};
 
-	refs = {};
+	constructor(props, context) {
+		super(props, context);
 
-	state = {
-		children: getChildMapping(this.props.children || [])
-	};
+		this.refs = {};
+
+		this.state = {
+			children: getChildMapping(this.props.children || [])
+		};
+
+		this.performLeave = this.performLeave.bind(this);
```

Comment:
Since Buble doesn't support property initializers, I had to bind the methods here :/